back-end-side/Models/Record.cs

```diff
@@ -20,7 +20,7 @@ public class Record : IComparable
         [Ignore]
         public int ID { get; set; }
         [Ignore]
-        public string? Category { get; set; } = "other";
+        public string Category { get; set; } = "other";
```

Comment:
why you removed question mark?

Reply:
because Category is never null since we assign "other" to it